Description: Disable doctests which require absent `testdata/`
Last-Update: 2023-03-02
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -48,3 +48,6 @@
     "alloc",
     "cfb",
 ]
+
+[lib]
+doctest = false
